src/index.js

```diff
@@ -44,6 +45,41 @@ class Index extends Component {
     this.onFinish = this.onFinish.bind(this);
   }
 
+  componentDidMount() {
+      let url_parameter = {};
+      const currLocation = this.props.location.search;
+      let parArr = currLocation.split("?");
+      if (parArr.length > 1) {
+          parArr = parArr[1].split("&");
+          for (let i = 0; i < parArr.length; i++) {
+              const parr = parArr[i].split("=");
+              if (parr[0] === 'lang') {
+                  for (let key of Object.keys(Localization.localizedStringList)) {
+                      if (parr[1] === key) {
```

Comment:
What if url has `#`? Just find our url is https://azure-samples.github.io/raspberry-pi-web-simulator/#Getstarted. It will be https://azure-samples.github.io/raspberry-pi-web-simulator/?lang=en-us#Getstarted, right?